client/me/billing-history/billing-history-table.jsx

```diff
@@ -46,17 +46,8 @@ module.exports = React.createClass( {
 				{ ...this.props }
 				initialFilter={ { date: { newest: 5 } } }
 				header
-				description={ function( transaction ) {
-					return (
-						<div className="transaction-links">
-							<a className="view-receipt" href={ '/me/billing/' + transaction.id } onClick={ this.recordClickEvent( 'View Receipt in Billing History' ) } >
-								{ this.translate( 'View Receipt' ) }
-							</a>
-							{ this.renderEmailAction( transaction.id ) }
-						</div>
-					);
-				}.bind( this ) }
-				/>
+				renderEmpty={ this.renderEmpty }
+				description={ this.renderDescription } />
```

Comment:
`description` is similarly a misnomer. Something like `transactionRenderer` would make more sense to indicate that we expect a function that takes a transition and spits out some React components.